appium-tests/android/android.spec.js

```diff
@@ -1,28 +1,34 @@
-/*jslint node: true, plusplus: true */
+/*jshint node: true */
 /*global beforeEach, afterEach */
 /*global describe, it, xit, expect, jasmine */
-'use strict';
 
 // these tests are meant to be executed by Cordova Medic Appium runner
 // you can find it here: https://github.com/apache/cordova-medic/
 // it is not necessary to do a full CI setup to run these tests
 // just run "node cordova-medic/medic/medic.js appium --platform android --plugins cordova-plugin-camera"
 
+'use strict';
+
 var wdHelper = require('../helpers/wdHelper');
 var wd = wdHelper.getWD();
 var cameraConstants = require('../../www/CameraConstants');
 var cameraHelper = require('../helpers/cameraHelper');
+var screenshotHelper = require('../helpers/screenshotHelper');
 
 describe('Camera tests Android.', function () {
+
+    var STARTING_MESSAGE = 'Ready for action!',
+        RETRY_COUNT = 3, // how many times to retry taking a picture before failing
+        MINUTE = 60 * 1000;
+
     var driver,
-        startingMessage = 'Ready for action!',
         // the name of webview context, it will be changed to match needed context if there are named ones:
         webviewContext = 'WEBVIEW',
-        // this indicates if device library has test picture:
+        // this indicates if device library has the test picture:
         isTestPictureSaved = false,
-        // this indecates that there was critical error and tests cannot continue:
+        // this indecates that there was a critical error and tests cannot continue:
         stopFlag = false,
-        // we need to know the screen width and height to properly click on the first image in the gallery
+        // we need to know the screen width and height to properly click on an image in the gallery
         screenWidth = 360,
         screenHeight = 567;
```

Comment:
No, these values can change when we determining the real device resolution.
These are just default values.